Index: mainQ.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import pars\r\nimport message\r\nimport vk_api\r\n\r\nfuck = pars.Parse(pars.Site('https://edu.donstu.ru/Rasp/Rasp.aspx?group=32353&sem=2'))\r\n\r\nvk_session = vk_api.VkApi(token=\"4e3f2af11078b879a7a574564ca5d14f3eb100cdb69c6ff953e1096c5a67f5a6d06233a295e52bc416026\")\r\n\"\"\"ddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd\r\nрпр\r\n\"\"\"\r\nmessage.Mess(fuck, vk_session)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- mainQ.py	(revision 0dc95a1a3e3184d425e69621bf4df230801c1624)
+++ mainQ.py	(date 1587933699527)
@@ -4,8 +4,12 @@
 
 fuck = pars.Parse(pars.Site('https://edu.donstu.ru/Rasp/Rasp.aspx?group=32353&sem=2'))
 
-vk_session = vk_api.VkApi(token="4e3f2af11078b879a7a574564ca5d14f3eb100cdb69c6ff953e1096c5a67f5a6d06233a295e52bc416026")
-"""ddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd
-рпр
+vk_session = vk_api.VkApi( token="4e3f2af11078b879a7a574564ca5d14f3eb100cdb69c6ff953e1096c5a67f5a6d06233a295e52bc416026")
+"""
+хуй соси если не наешл этот писька комит 
+
+а Александр молодец
+
+Хорошо сосет хуи Олег
 """
 message.Mess(fuck, vk_session)
